.github/workflows/ci.yml

```diff
@@ -27,3 +27,19 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+      - name: Generate README
+        uses: docker://pandoc/core:2.18
```

Comment:
Do we want to specify the patch release too?

Reply:
I would hope that they maintain output compatibility across patch releases

Reply:
Okay, leaving as-is then.